packages/eds-core-react/src/components/Slider/Slider.test.tsx

```diff
@@ -44,6 +45,18 @@ describe('Simple slider', () => {
     )
     expect(asFragment()).toMatchSnapshot()
   })
+  it('Should pass a1yy test', async () => {
```

Comment:
*a11y

Btw you probably don't have to use act unless you get the console warning about it. I don't know if you are getting it here or not